integration/index_test.go

```diff
@@ -80,8 +80,8 @@ func prepareSociIndices(t *testing.T, sh *dockershell.Shell) []testImageIndex {
 	return testImages
 }
 
-func getZtocDigests(sh *dockershell.Shell, img imageInfo) ([]string, error) {
-	ztocInfoBytes := sh.O("soci", "ztoc", "list")
+func getZtocDigestsForImage(sh *dockershell.Shell, img imageInfo) ([]string, error) {
```

Comment:
what if the image has multiple soci indices generated? Will that break any tests?

Reply:
We are only using one platform for each image, so there should only be one index I think? Regardless, It shouldn't as I am only testing one ztoc digest anyway.

Reply:
>  what if the image has multiple soci indices generated

It does not. For the integ test, we pull image for only 1 platform 
https://github.com/awslabs/soci-snapshotter/blob/bff4b07408aa3df86d5932a036880c5bf2ff6f4a/integration/testutil_soci.go#L63

Reply:
It doesn't except where we test multi-platform create
https://github.com/awslabs/soci-snapshotter/blob/main/integration/create_test.go#L122-L131

Maybe there's value in adding a `platform` flag to `ztoc list` at some point.